app/packages/core/src/components/Modal/ImaVidLooker.tsx

```diff
@@ -210,6 +257,10 @@ export const ImaVidLookerReact = React.memo(
       name: timelineName,
       config: timelineCreationConfig,
       waitUntilInitialized: readyWhen,
+      // using this mechanism to resume fetch if it was paused
+      // ideally we have control of fetch in this component but can't do that yet
```

Comment:
Is the grid moving to react? :)

Reply:
I definitely haven't thought about that one way or the other. 😅